.github/workflows/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+### Describe your changes
+
+## Issue ticket number and link
+
+### Checklist before requesting a review
+#### General
+- [ ] I have performed a self-review of my code
+- [ ] The commit message follows our guidelines
+- [ ] I have put the PR link into the planner
+- [ ] The PR has a clear and descriptive title and a detailed description of the changes
```

Comment:
I would put this more like - the commit title follows conventional commit standards. The title of the PR is what will show up in the commit history and we should make sure it follows a standard. We could go as far as to enforce it, but with just the two of you, I think it might be overkill

Reply:
Okay sounds good, will make those changes